app/src/main/java/co/nimblehq/compose/crypto/ui/components/chartintervals/ChartIntervalsButton.kt

```diff
@@ -0,0 +1,91 @@
+package co.nimblehq.compose.crypto.ui.components.chartintervals
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import co.nimblehq.compose.crypto.ui.theme.Color
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp12
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp14
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp4
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp45
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp8
+import co.nimblehq.compose.crypto.ui.theme.Style
+
+@Composable
+fun ChartIntervalsButtonGroup(
+    modifier: Modifier,
+    onIntervalChanged: (TimeIntervals) -> Unit
+) {
+
+    val selectedColor = remember { mutableStateOf(0) }
+    Row(
+        modifier = modifier,
+        horizontalArrangement = Arrangement.spacedBy(Dp14)
+    ) {
+        val listOfInterval = listOf(
+            TimeIntervals.ONE_DAY,
+            TimeIntervals.ONE_WEEK,
+            TimeIntervals.ONE_MONTH,
+            TimeIntervals.ONE_YEAR,
+            TimeIntervals.FIVE_YEAR
+        )
+
+        listOfInterval.forEachIndexed { index, interval ->
```

Comment:
how about using `value()`?

```suggestion
        TimeIntervals.values().forEachIndexed { index, interval ->
```

Reply:
@luongvo Nice 💯